src/Expectation.php

```diff
@@ -60,6 +60,17 @@ public function and($value): Expectation
         return new self($value);
     }
 
+    /**
+     * Parses Json String to Array.
+     */
+    public function json(): Expectation
+    {
+        Assert::assertIsString($this->value);
```

Comment:
Could this be refactored to `return $this->toBeJson()->and(json_decode($this->value, true));`?

Reply:
ahah.